pkg/doc/verifiable/example_credential_test.go

```diff
@@ -103,30 +73,31 @@ func ExampleCredential_embedding() {
 	// Marshal to JSON to verify the result of decoding.
 	vcBytes, err := json.Marshal(vc)
 	if err != nil {
-		fmt.Println("failed to marshal VC to JSON")
+		panic("failed to marshal VC to JSON")
 	}
 
 	fmt.Println(string(vcBytes))
 
 	// Marshal to JWS.
 	jwtClaims, err := vc.JWTClaims(true)
 	if err != nil {
-		fmt.Println(fmt.Errorf("failed to marshal JWT claims of VC: %w", err))
+		panic(fmt.Errorf("failed to marshal JWT claims of VC: %w", err))
 	}
 
 	jws, err := jwtClaims.MarshalJWS(verifiable.EdDSA, getEd25519Signer(issuerPrivKey), "")
 	if err != nil {
-		fmt.Println(fmt.Errorf("failed to sign VC inside JWT: %w", err))
+		panic(fmt.Errorf("failed to sign VC inside JWT: %w", err))
 	}
 
 	fmt.Println(jws)
 
 	// Parse JWS and make sure it's coincide with JSON.
 	vcParsed, err := verifiable.ParseCredential(
 		[]byte(jws),
-		verifiable.WithPublicKeyFetcher(verifiable.SingleKey(issuerPubKey, kms.ED25519)))
+		verifiable.WithPublicKeyFetcher(verifiable.SingleKey(issuerPubKey, kms.ED25519)),
+		verifiable.WithJSONLDDocumentLoader(getJSONLDDocumentLoader()))
 	if err != nil {
-		fmt.Println(fmt.Errorf("failed to encode VC from JWS: %w", err))
+		panic(fmt.Errorf("failed to encode VC from JWS: %w", err))
```

Comment:
we you change it to panic

Reply:
Just because in many cases there is no more sense to continue the test execution. The same approach is used e.g. at https://github.com/hyperledger/aries-framework-go/blob/master/pkg/client/outofband/example_test.go